### DISCUSSION TITLE
Manual Mesh Bed Leveling

### DISCUSSION THREAD

**Discussion by RUtarah** [Jyers/Marlin#306] [Q&A] [answered]
I just print a large base of my design(almost full bed size). The 1st layer was absolutely working perfect, but for the next layers my nozzle start griping the previous layer and scratch it. And I can hear scratching sound from nozzle touching the top of print part. Just want to know anyone got this happen? Is it the manual mesh bed leveling perform only just for 1st layer?

**Comment by Jyers** [comment, marked as answer]
Manual mesh is applied by default for the first 10 mm, so it should go far past the first layer. Are you still experiencing the issue?

**Reply by RUtarah** [reply]
yes and how to overcome this problem ya? now I only use manual level by paper

**Reply by Jyers** [reply]
Sounds like perhaps it could be a bit of over-extrusion or something of the like. If not you can try with the mesh disabled to see if you get a different result.

**Reply by RUtarah** [reply]
When I disable manual mesh bed leveling it much better no gripping or scratching again. Like my mesh not perform but I already re flashed the firmware.

**Reply by jetel73** [reply]
If your first layer is not shiny and clean maybe it is over extrusion, maybe your level is unperfect et/or your mesh is not well set and/or not working properly. You probably already know what to do but maybe it could help to compare with my procedure to avoid this problem. 

- Flash your printer with the 5x5 Manual Mesh levelling 1.2.2 firmware
- Go to your slicer's beginning gcode and add the M420 S1 Z2 command just after the G28 like this (Cura for me):

> Ender 3 Custom Start G-code
> G92 E0 ; Reset Extruder
> G28 ; Home all axes
> M420 S1 Z2 ; Restore manual mesh level and set fade height to 2mm 
> G1 Z2.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed
> G1 X0.1 Y20 Z0.3 F5000.0 ; Move to start position
> G1 X0.1 Y200.0 Z0.3 F1500.0 E15 ; Draw the first line

- Level your bed the best you can with the manual levelling four corners procedure by adjusting the bed knobs. You'd better do crosses from corner to corner instead of square. square is good for control at the end in my opinion.
- Do your mesh procedure by moving your piece of paper while maintaining the knob on babystep down, hearing the bips while the head is moving down. When you feel that the head is just in contact release the knob and move to the other point to reach the same contact feeling. If you're not happy with your adjustment, raise your head 30 and start the procedure again. Do not abuse Up and Down adjustments to create the mesh or it will be inaccurate. Doing your adjustment at once it's the best.

You should have great results. 
If you stick a piece of PLA as a needle, on the top of your Z screw with some Blu Tack or some tape, you will notice that the correction is gradually decreased until the printer reaches 2mm. 
If you're still not satisfied maybe you have to set your extruder step/mm.
Good luck
